notification_center.py

```diff
@@ -50,15 +50,18 @@ def notify(data, buffer, date, tags, displayed, highlight, prefix, message):
 	# passing `None` or `''` still plays the default sound so we pass a lambda instead
 	sound = weechat.config_get_plugin('sound_name') if weechat.config_get_plugin('sound') == 'on' else lambda:_
 	activate_bundle_id = weechat.config_get_plugin('activate_bundle_id')
+        group_prefix = 'weechat.nc'
```

Comment:
Why not just `group_prefix = 'weechat'`?

And it shouldn't be indented.

Reply:
'weechat` would work, but I decided to scope it further to this specific plugin. Seemed like a good idea at the time, but perhaps it's inconsequential? Thoughts?

Reply:
Let's go with just `weechat`. If other weechat plugins use the notification center, although I highly doubt anyone would have multiple plugins using it, they should be grouped together regardless.